### ISSUE TITLE
Investigate how `cargo new` collides with `cargo ws new`

### ISSUE THREAD

**Issue by pksunkara** [pksunkara/cargo-workspaces#144]
MSRV needed 1.75.0

https://github.com/rust-lang/cargo/pull/12779

**Comment by filleduchaos** [comment]
Yeah this breaks `cargo ws create` - the package creation itself succeeds, but because the output from Cargo does not contain "Created" [cargo workspaces assumes it failed](https://github.com/pksunkara/cargo-workspaces/blob/ee077fae4efdff41b255ba5437f9bd2926a14b57/cargo-workspaces/src/create.rs#L180-L182)

Std output of `cargo new` in a workspace (on nightly 2024-03-02):

```
    Creating library `foo` package
      Adding `foo` as member of workspace at `/some/path/to/workspace`
```

**Comment by sushruth** [comment]
I opened a PR for it potentially fixing this - #152
